tests/integration/test_integration.py

```diff
@@ -37,7 +38,9 @@ async def build_and_deploy(ops_test):
         application_name=AMF_CHARM_NAME,
         channel="edge",
         trust=True,
+        config={"external-amf-ip": "192.168.70.132", "external-amf-hostname": "amf"},
```

Comment:
why are the external host and ip needed? isn't the hostname of AMF in the cluster enough?

Reply:
It seems external host and ip are optional. The information that provided by cluster supposed to be enough.

Reply:
You are right. There is no need to add the external host and IP. I removed this line